README.md

```diff
@@ -4,17 +4,42 @@ Welcome. This project is meant to accompany you during your interview at Airalo.
 ## Requirements
 - PHP >= 8.0
 
-### Installation
+## Installation
 ```
 cd [YOUR WORKSPACE]
 mkdir -p airalo-assessment/project   
 cd airalo-assessment/project
 git clone git@github.com:Airalo/backend-assessment.git .
 ```
 
-#### Composer dependencies
+## Local PHP + DATABASE solution
 ```
+## IF YOU HAVE A MYSQL SERVER
 composer install
+cp .env.example .env
+php artisan key:generate
+## edit your db connection
+```
+
+```
+## IF YOU DON'T HAVE A MYSQL SERVER
+composer install
+cp .env.sqlite .env && cp database/database.sqlite.example database/database.sqlite
```

Comment:
The `database/database.sqlite.example` file does not exists in the repo
This is ignored in `database/.gitignore` file

I ran the `migrate` command without it and there were no problems.

```
php artisan migrate

   WARN  The SQLite database does not exist: database/database.sqlite.  

  Would you like to create it? (yes/no) [no]
❯ yes

   INFO  Preparing database.  

  Creating migration table ................................................................................................................ 5ms DONE

   INFO  Running migrations.  

  2014_10_12_000000_create_users_table .................................................................................................... 2ms DONE
  2014_10_12_100000_create_password_resets_table .......................................................................................... 1ms DONE
  2019_08_19_000000_create_failed_jobs_table .............................................................................................. 1ms DONE
  2019_12_14_000001_create_personal_access_tokens_table ................................................................................... 1ms DONE

```

Reply:
I added `database.sqlite.example` file and updated the `.gitignore` file inside the database folder.